sbt-scalajs-bundler/src/main/scala/scalajsbundler/WebpackDevServer.scala

```diff
@@ -47,28 +45,26 @@ private [scalajsbundler] class WebpackDevServer {
   }
 
   private class Worker(
-    npmDir: File,
     workDir: File,
     configPath: File,
     port: Int,
     extraArgs: Seq[String],
     logger: ProcessLogger
   ) {
 
-    logger.info("Starting webpack-dev-server");
-
-    val devServerPath = npmDir / "node_modules" /
-    "webpack-dev-server" / "bin" / "webpack-dev-server.js"
-
     val command = Seq(
       "node",
-      devServerPath.absolutePath,
+      "node_modules/webpack-dev-server/bin/webpack-dev-server.js",
       "--config",
       configPath.absolutePath,
       "--port",
       port.toString
     ) ++ extraArgs
 
+    logger.info("Starting webpack-dev-server");
+    command.foreach(arg => logger.info("  " + arg))
+    logger.info("workDir: " + workDir.toString)
```

Comment:
Do we want to keep all of these logs at the `info` level? I would put the two last lines at the `debug` level.